docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -12,7 +12,7 @@ information.
 
 All data tables in SilverStripe are defined as subclasses of [api:DataObject]. The [api:DataObject] class represents a 
 single row in a database table, following the ["Active Record"](http://en.wikipedia.org/wiki/Active_record_pattern) 
-design pattern. Database Columns are defined as [Data Types](data_types_and_casting) in the static `$db` variable 
+design pattern. Database Columns are is defined as [Data Types](data_types_and_casting) in the static `$db` variable 
```

Comment:
Something wrong here, it's a reverse of https://github.com/silverstripe/silverstripe-framework/pull/4000/files